tracer/src/Datadog.Trace.Coverage.collector/CoverageCollector.cs

```diff
@@ -48,7 +50,7 @@ public override void Initialize(XmlElement configurationElement, DataCollectionE
                 };
                 events.TestCaseStart += (sender, args) =>
                 {
-                    _logger?.Debug($"Test case start: {args.TestCaseName}");
+                    _logger?.Debug($"Test case start [{Interlocked.Increment(ref _testNumber)}]: {args.TestCaseName}");
```

Comment:
This seems unrelated, is this supposed to be here?

Reply:
it's unrelated but is a micro change to have the number of tests in the logs.